src/LoginPage.test.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import { MemoryRouter } from 'react-router-dom';
+
+import { render } from '@testing-library/react';
+
+import LoginPage from './LoginPage';
+
+describe('LoginPage', () => {
+  const renderLoginPage = () => render(
+    <MemoryRouter>
+      <LoginPage />
+    </MemoryRouter>,
+  );
+
+  it('renders title', () => {
+    const { container } = renderLoginPage();
+
+    expect(container).toHaveTextContent('Log In');
+  });
+
+  it('renders input controls', () => {
+    const { container } = renderLoginPage();
+
+    expect(container).toHaveTextContent('E-mail');
```

Comment:
```suggestion
    expect(getByLabelText('E-mail')).not.toBeNull();
```
input은 label을 통해서 element 확인을 하면 좀 더 확실하겠네요.